railties/lib/rails/test_unit/reporter.rb

```diff
@@ -17,6 +17,8 @@ def record(result)
       end
 
       if output_inline? && result.failure && (!result.skipped? || options[:verbose])
+        result.failures.each { |f| f.backtrace.clear } if result.error?
+
```

Comment:
I _really_ don't like this change. Please read the comment on the commit for my thoughts.

Reply:
Given how `SuppressedSummaryReporter` works, that means when you run tests with `inline` on and you have an error, you NEVER get a backtrace? That seems bad.

Reply:
This change is causing problems in master.  I've got some failing tests in master and with this change the backtrace is gone.  Without the change I can see the backtrace.  Why do we need this change?